x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/predicate/operator/comparison/Comparisons.java

```diff
@@ -19,6 +19,11 @@ public static Boolean eq(Object l, Object r) {
         return i == null ? null : i.intValue() == 0;
     }
 
+    static Boolean neq(Object l, Object r) {
+        Integer i = compare(l, r);
+        return i == null? null : i.intValue() != 0;
```

Comment:
`return i == null? null : i.intValue() != 0;` just add a whitespace: `return i == null ? null : i.intValue() != 0;`. If the PR is held back by this picky comment, you can ignore it and merge.

Reply:
no worries, I'll fix, need to merge once the select-csv.spec is fixed anyway.